src/loaders/mod.rs

```diff
@@ -28,3 +29,4 @@ pub use self::sprite::*;
 pub use self::stream::ByteStream;
 pub use self::texture::TextureLoader;
 pub use self::version::{InternalVersion, MajorFirst, MinorFirst, Version};
+pub use self::xml::*;
```

Comment:
Give that there are only `Connection` and `XMLFileLoader` here to export, I would prefer if we remove the wildcard export.